templates/registration/login.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h2>Login</h2>
+  {% if form.errors %}
+    <p style="color: blue">Invalid credentials</p>
```

Comment:
The error message color is set to blue, which might not be the best choice for indicating errors. Consider using a more conventional color like red to make it clear that it's an error message.